### ISSUE TITLE
Prevent duplicate equations in E1 (and others?)

### ISSUE THREAD

**Issue by siwelwerd** [TeamBasedInquiryLearning/checkit-tbil-la#14]
Unsure if this should be handled at the outcome level, or modifying the nice matrix generating function

<img width="683" alt="Screenshot 2021-02-03 123525" src="https://user-images.githubusercontent.com/30658947/106792921-62583f00-661c-11eb-9b4d-799acaaca5e1.png">

**Comment by StevenClontz** [comment]
Yeah I think it should at least be an option on the matrix generator to not repeat rows (or allow 0=1).

Leaving this open until https://github.com/StevenClontz/checkit/issues/29 is implemented and we re-build exercises.
